python/ray/workers/default_worker.py

```diff
@@ -51,11 +51,6 @@
     type=str,
     default=ray_constants.LOGGER_FORMAT,
     help=ray_constants.LOGGER_FORMAT_HELP)
-parser.add_argument(
-    "--collect-profiling-data",
-    type=int,  # int since argparse can't handle bool values
-    default=1,
-    help="Whether to collect profiling data from workers.")
```

Comment:
Now that we flush the GCS (including the profiling table), we no longer need this.